functions.py

```diff
@@ -0,0 +1,63 @@
+import time
+from objects import *
+
+
+# score_a = 0
+# score_b = 0
+
+# rockets movement
```

Comment:
Leftover comments are not appreciated in production code. Those are usually removed prior to merge into main branch.